runner/execer/os/os_execer.go

```diff
@@ -15,13 +15,23 @@ func NewExecer() execer.Execer {
 
 type osExecer struct{}
 
+type WriterDelegater interface {
```

Comment:
Maybe copy the comment from output.go.

How will this be used as we complete milestones? Atm we want WorkerServer to save to the cloud and we'll probably want to control batching lines, tee'ing, etc, and we wouldn't implement WriterDelegate for that I think. Then for scootfs, we'll either be writing to a file and using WriterDelegate, or writing directly to manhattan/whatever without WriterDelegate. Are we sure this will be an ongoing useful construct?